tests/test_my_projects.py

```diff
@@ -83,6 +85,16 @@ def test_custom_collection(self, driver, session, default_project, my_projects_p
         WebDriverWait(driver, 5).until(EC.text_to_be_present_in_element((By.CSS_SELECTOR, 'li[data-index="4"] span'), '(1)'))
         assert '1' in my_projects_page.first_custom_collection.text
 
+        osf_api.delete_custom_collections(session)
+
+    def test_delete_custom_collection(self, session, driver, my_projects_page):
+        # API Setup
+        osf_api.delete_custom_collections(session)
```

Comment:
We probably want to separate the roles for setup and teardown. If teardown clears the database, we don't need to repeat that in setup. However, this only works in an ideal scenario where every test does its job of teardown for the next one. Any thoughts?

Reply:
Exactly, I put a pre-delete in case a test fails and never makes it to the cleanup part on the previous run. Should we just do pre-deletes instead of before and after?